large_page-c/lp_preload.c

```diff
@@ -1,7 +1,49 @@
+#define _GNU_SOURCE
+#include <link.h>
 #include <stdio.h>
 #include <stdlib.h>
+#include <string.h>
+#include <sys/types.h>
+#include <regex.h>
 #include "large_page.h"
 
+void printErr (map_status status, const char * lib)
+{
```

Comment:
nipick. Please put open curly braces on the same line as function definition matching with rest of the coding standard.